### PR TITLE
Survival analysis tab enhancement

### PR DESCRIPTION
**Pull request by msarrias**
Added new functionalities to the "Survival Analysis" and "select target gene" tabs - read commit messages below. 

commit [cf14849](https://github.com/collaborativebioinformatics/VariantSurvival/pull/17/commits/cf1484918ef9be24b540ce97212fdc1adce95501): fixed a bug in the KM plot. 

All new input fields need to be relocated (to make it look pretty), I'll leave that for Hiba.

### CODE REVIEW

**Comment by msarrias** [comment]
Here some screenshots:

New "summary" tab ( any re-naming suggestion?)
image

Dropped the zero labels:

image

Added an input field for including more columns:

image

Added more options to customize the KM plot, I'm not entirely convinced by it, as when any option is selected/removed the plot doesn't update, instead, it computes the entire thing all over again. Not sure how to change it but it works.

image

**Comment by Krannich479** [comment]
Here are things I observed when installing and running the _msarrias_ branch:
- When I tried to start the app I was asked to install "shinyjs". Is this a new dependency? Should this be among the auto-installed dependencies? Is it just me facing this artifact?
- When I try to use a min/max number of SVs in the Survival Analysis tab than the app crashes (in Rstudio as well as in browser mode).
- In the Select Target Gene tab -> Table I would call it "SV count in gene of interest" over "SV count per gene"

File: VariantSurvival_package/R/VariantSurvival.R [hunk 1]
```diff
@@ -337,6 +520,7 @@ VariantSurvival <- function(vcffile, metadatafile,demo){
 ############# Helper functions ###################
 install_load_requirements<- function() {
   if (!require("shiny")) install.packages("shiny")
+  if (!require("shiny")) install.packages("shinyjs")
```
Comment (msarrias) [inline VariantSurvival_package/R/VariantSurvival.R:523]: @Krannich479 ohhh nice catch.

**Comment by msarrias** [comment]
@Krannich479, try again!

**Comment by Krannich479** [comment]
`if (!require("shiny")) install.packages("shinyjs")`

Confirmed, new auto-load of _shinyjs_ dependency works!

**Review by Krannich479 — COMMENTED** [review]
New features work. PR approved

**Review by Krannich479 — APPROVED** [review]

**Pull request merged**